etc/compile_check.sh

```diff
@@ -2,40 +2,36 @@
 set -e # exit when any command fails
 set -x # show all commands being run
 
-GC=go
+GC=go$GO_VERSION
```

Comment:
Should this default to 1.18 for local usage?

Reply:
That's a great idea! Will do.